share/doc/xml/specextract.xml

```diff
@@ -1891,6 +1891,17 @@ unix% cat output.lis
 	refcoord parameter is not set, otherwise the script will
 	error out.
       </PARA>
+      <PARA>
+	When using a blanksky or stowed background file for background
+	spectrum extraction where the OBS_ID keyword is included would
+	fail with 4.15.0 which has been re-worked.
+      </PARA>
+      <PARA>
+	Also, a bug where an incomplete specextract history written to
+	the output files when using TTY screen output and a bug where
+	HRC background responses were created even with 'bkgresp=no'
+	have been fixed.
+      </PARA>
```

Comment:
How about, for these two paragraphs, we replace with something like

```
<PARA>
A number of bugs added in 4.15.0 have been fixed: the script history is now written out correctly when run intereactively,  the presence of the OBS_ID keyword in blanksky or stowed background files is no-longer a problem, and the script no-longer creates a HRC background response when 'bkgresp=no' is used.
</PARA>
```

(I think the HRC issue is technically not a 4.15.0 problem but not worth separating out, I think).